js/views/ConnectedPeersPage.js

```diff
@@ -11,6 +12,48 @@ export default class extends baseVw {
     }
 
     this.peers = options.peers;
+    this.peersLoaded = 0;
+    this.peersIterator = 12;
+  }
+
+  className() {
+    return 'userPage';
+  }
+
+  events() {
+    return {
+      'click .js-morePeersBtn': 'loadPeers',
+    };
+  }
+
+  get $peerWrapper() {
+    return this._$peerWrapper ||
+        (this._$peerWrapper = this.$('.js-peerWrapper'));
+  }
+
+  get $morePeers() {
+    return this._$morePeers ||
+        (this._$morePeers = this.$('.js-morePeers'))
+  }
+
+  loadPeers() {
+    if (this.peers.length > this.peersLoaded) {
+      this.peers.slice(this.peersLoaded, this.peersLoaded + this.peersIterator).forEach((peer) => {
+        console.log(peer)
+        const user = this.createChild(userShort, {
+          guid: peer,
+        });
+        this.$peerWrapper.append(user.render().$el);
+      });
+      this.peersLoaded += this.peersIterator;
```

Comment:
There's a good chance the last page will not be a full page, so for peersLoaded to be accurate, you would want to add the amount you sliced above.

But... it is inconsequential here. For what we are using peersLoaded for, it doesn't matter if it's wrong, as long as it's over not under.